gitgud/__main__.py

```diff
@@ -21,7 +21,7 @@ class InitializationError(Exception):
 
 class GitGud:
     def __init__(self):
-        self.file_operator = get_operator()  # Only gets operator if in a valid gitgud repo
+        self.file_operator = get_operator()  # Only gets operator if in a valid git
```

Comment:
You said you'd change this back to the original text in your previous PR, but I don't see those changes, can you push them?